frontend/src/concepts/topology/PipelineVisualizationSurface.scss

```
@@ -0,0 +1,8 @@
+.pipeline-visualization.m-is-open {
+  .pf-topology-container {
+    overflow-y: hidden;
+    .pf-v5-c-drawer__panel.pf-m-resizable {
+      min-width: 350px;
```

Previous version set the minimum with of the side panel to 500px, this fixes the hidden tab issue.
```suggestion
      min-width: 500px;
```

---

Thank you for this awesome suggestion @jeff-phillips-18 🎉